src/tutorials/quickstart.rst

```diff
@@ -1,34 +1,35 @@
-===============
-Running a build
-===============
+===========================
+Quickstart - Zika evolution
+===========================
```

Comment:
My prior thinking on this doc's place in our "learning ramp", and thus its title, is in 3a8ca8bb812ccba58c34131f8b1de2f079e806f6. I still suggest avoiding the "quickstart" moniker and keeping the title descriptive of what the tutorial shows how to do. Ok to include Zika in the title, but also I think its an incidental detail borne of convenience rather than an important choice and don't think it's necessary to call out. Good to align with the next doc in the ramp, however, as you've done.

So perhaps _Running a pathogen workflow_ and _Creating a pathogen workflow_?

To then align the TB-based tutorial's name, it could be _Creating a pathogen workflow (VCF)_ maybe?

Reply:
> So perhaps Running a pathogen workflow and Creating a pathogen workflow?

This is nice, and we should emphasise that these are _the same_ workflow, with the former deliberately ignoring the individual steps in the pipeline and the latter explaining each individual step.

> To then align the TB-based tutorial's name

In a previous meeting we decided to [Push TB down the (sidebar) order](https://docs.google.com/document/d/1HxnvQew-2y3ZIB4LWPlHle3nJGQiDTS0uP1uxoAtAuI/edit#), which I think could be done in this PR. I'd suggest a name of _Creating a bacterial pathogen workflow_. (That tutorial also needs some updating, but that's another issue!)

Reply:
Yep, we should definitely setup redirects. The process for setting them up is [documented in this repo's readme](https://github.com/nextstrain/docs.nextstrain.org#configuring-redirects).

(Ideally we'd maintain redirects in a file in this repo and use the RTD v3 API to keep the live redirects automatically synced with the file, but that's not yet reality.)

Reply:
> Does bacterial pathogen workflow here simply mean VCF in place of FASTA, or is it more complicated than that?

The TB workflow is more of an introductory bacterial workflow, in the same way the zika tutorial is an introductory viral workflow. As well as VCF inputs, it includes inference of anti-microbial resistance.

Reply:
@jameshadfield Ok, I assume your title will be more applicable to our readers. Updated and renamed file: 625640f2628b6b2386be2f03e97c2f258868235a

Also reordered sidebar as suggested: 2b9d03b895caca6171b12d0656b1165dc656c6e0